[*] Running ...
[*] Dataset loaded: /home/dokhanhnam1199/QD/problems/cvrp_aco/dataset/train50_dataset.npy with 10 instances.
/home/dokhanhnam1199/.local/lib/python3.10/site-packages/numpy/_core/fromnumeric.py:3860: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/home/dokhanhnam1199/.local/lib/python3.10/site-packages/numpy/_core/_methods.py:145: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
[*] Instance 0: 10.465754264272892
[*] Instance 1: 10.32887936084965
[*] Instance 2: 11.18125259909633
[*] Instance 3: 11.422774767019213
[*] Instance 4: 11.43542273282365
[*] Instance 5: 9.843626862154688
[*] Instance 6: 10.440643717726728
[*] Instance 7: 11.648016304619151
[*] Instance 8: 9.988958466636308
[*] Instance 9: 10.399828550980006
[*] Average:
10.715515762617862
64
19
1106.2140488035468
